cfgmgr/portmgr.cpp

```diff
@@ -205,30 +256,37 @@ void PortMgr::doTask(Consumer &consumer)
             {
                 writeConfigToAppDb(alias, field_values);
             }
-
             if (!portOk)
             {
                 SWSS_LOG_INFO("Port %s is not ready, pending...", alias.c_str());
+                writeConfigToAppDb(alias, "mtu", mtu);
+                writeConfigToAppDb(alias, "admin_status", admin_status);
+                writeConfigToAppDb(alias, "dhcp_rate_limit", dhcp_rate_limit);
 
                 /* Retry setting these params after the netdev is created */
                 field_values.clear();
                 field_values.emplace_back("mtu", mtu);
                 field_values.emplace_back("admin_status", admin_status);
+                field_values.emplace_back("dhcp_rate_limit", dhcp_rate_limit);
+
                 it->second = KeyOpFieldsValuesTuple{alias, SET_COMMAND, field_values};
                 it++;
                 continue;
             }
-
             if (!mtu.empty())
             {
                 setPortMtu(alias, mtu);
                 SWSS_LOG_NOTICE("Configure %s MTU to %s", alias.c_str(), mtu.c_str());
             }
-
             if (!admin_status.empty())
             {
                 setPortAdminStatus(alias, admin_status == "up");
                 SWSS_LOG_NOTICE("Configure %s admin status to %s", alias.c_str(), admin_status.c_str());
+            }            
+            if (!dhcp_rate_limit.empty())
+            {
+                setPortDHCPMitigationRate(alias, dhcp_rate_limit);
```

Comment:
This should be set only if there is explicit configuration of `dhcp_rate_limit`.  It should stay the same as today (no kernel TC) if there is no config.